src/components/Statistics/FeedbackItem.vue

```diff
@@ -72,9 +80,16 @@ export default {
       const { about: { store: { name } = {} } = {} } = this.feedback
       return name
     },
+    storeId () {
+      const { about: { store: { id } = {} } = {} } = this.feedback
+      return id
+    },
     userName () {
       return this.feedback && this.feedback.givenBy && this.feedback.givenBy.displayName
     },
+    userId () {
+      return this.feedback && this.feedback.givenBy && this.feedback.givenBy.id
```

Comment:
I think we need a nicer way to resolve properties without throwing errors, maybe I'll add a helper soon...